client/internal/acl/manager_create.go

```diff
@@ -18,10 +18,7 @@ func Create(iface IFaceMapper) (manager *DefaultManager, err error) {
 		if err != nil {
 			return nil, err
 		}
-		return &DefaultManager{
-			manager:    fm,
-			rulesPairs: make(map[string][]firewall.Rule),
-		}, nil
+		return newDefaultManager(fm), nil
```

Comment:
"github.com/netbirdio/netbird/client/firewall" imported and not used in this file, causing CI/CD to fail

Reply:
Fixed